src/main/java/md/miller1995/Dealership/models/Car.java

```diff
@@ -0,0 +1,82 @@
+package md.miller1995.Dealership.models;
+
+import java.time.LocalDate;
+
+public class Car {
+    private long id;
+    private String brand;
+    private String model;
+    private String vin;
+    private LocalDate dateOfManufactured;
+    private String color;
+    private float price;
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    public String getBrand() {
+        return brand;
+    }
+
+    public void setBrand(String brand) {
+        this.brand = brand;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public void setModel(String model) {
+        this.model = model;
+    }
+
+    public String getVin() {
+        return vin;
+    }
+
+    public void setVin(String vin) {
+        this.vin = vin;
+    }
+
+    public LocalDate getDateOfManufactured() {
+        return dateOfManufactured;
+    }
+
+    public void setDateOfManufactured(LocalDate dateOfManufactured) {
+        this.dateOfManufactured = dateOfManufactured;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public float getPrice() {
+        return price;
+    }
+
+    public void setPrice(float price) {
+        this.price = price;
+    }
+
```

Comment:
maybe, it is necessary to use the Lombok dependency to automatically generate (@ Getter, @ Setter, @ ToString)

Reply:
in this case I generated setter, getter and toString just for practice and skill, in future classes I will use Lombok.